app/frontend/components/pages/login/mnemonicAuth.js

```diff
@@ -75,7 +76,7 @@ class LoadByMenmonicSectionClass extends Component {
                 mnemonic && !mnemonicValidationError ? 'pulse' : ''
               }`,
               disabled: !mnemonic || mnemonicValidationError,
-              onClick: () => loadWallet({cryptoProvider: 'mnemonic', secret: mnemonic}),
+              onClick: () => loadWallet({cryptoProvider: MNEMONIC, secret: mnemonic}),
```

Comment:
I am not sure `MNEMONIC` is a good name of `cryptoProvider` as mnemonic does not provide crypto. @refi93 any suggestions on naming?

Reply:
to keep it consistent with module names, I would name it WALLET_SECRET

Reply:
and I agree with namespacing those constants as you pointed out below in crypto-provider-factory.js